### PR TITLE
V4

### PR DESCRIPTION
**Pull request by Nannk**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [x] Feature
- [ ] Documentation update

## Description

Steam Deck install script

## Checklist

- [ ] any anime playing
- [ ] bumped version
- [ ] next, prev and replay work
- [ ] quality works
- [ ] downloads work
- [ ] quality works with downloads
- [ ] select episode -a and rapid resume work
- [ ] syncplay -s works
- [ ] autoplay, aka range selection, works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,6 +204,96 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd ~ 
+cp ~/.ani-cli/ani-cli ~/.local/bin/
+chmod +x ~/.local/bin/ani-cli
+```
+
+##### Optional: add desktop entry:
+
+```
+touch ~/.local/share/applications/ani-cli.desktop
```
Comment (justchokingaround) [inline README.md:283]: touch isn't required if u'll echo in it

File: README.md [hunk 2]
```diff
@@ -203,6 +204,96 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+cp ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.local/bin/
+chmod +x ~/.local/bin/aria2c
+```
+
+##### Install ani-cli:
+
+```
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd ~
```
Comment (justchokingaround) [inline README.md:275]: `cd` is enough, no need for `cd ~`

File: README.md [hunk 3]
```diff
@@ -203,6 +204,96 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+~/.fzf/install
+
+mkdir ~/.aria2c
+wget -O ~/.aria2c/aria2-1.36.0.tar.bz2 https://github.com/q3aql/aria2-static-builds/releases/download/v1.36.0/aria2-1.36.0-linux-gnu-64bit-build1.tar.bz2
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+cp ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.local/bin/
+chmod +x ~/.local/bin/aria2c
+
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd ~
```
Comment (justchokingaround) [inline README.md:229]: `cd` is enough, no need for `cd ~`

File: README.md [hunk 4]
```diff
@@ -253,6 +344,17 @@ pkg remove ani-cli
 ```sh
 rm "$PREFIX/bin/ani-cli"
 ```
+* Steam Deck
+```
+rm "~/.local/bin/ani-cli"
```
Comment (justchokingaround) [inline README.md:349]: isn't 
```sh
rm -rf ~/.ani-cli/
```
also missing?

**Review by justchokingaround — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Nannk) [reply README.md:349]: hmm, yes

File: README.md (hunk 1 above)
Comment (Nannk) [reply README.md]: deleted that line

File: README.md (hunk 2 above)
Comment (Nannk) [reply README.md]: https://github.com/pystardust/ani-cli/pull/976/commits/c099e3c7a497f834c9f3cce110339f1f76219860

File: README.md (hunk 3 above)
Comment (Nannk) [reply README.md]: https://github.com/pystardust/ani-cli/pull/976/commits/c099e3c7a497f834c9f3cce110339f1f76219860

File: README.md [hunk 5]
```diff
@@ -203,6 +204,93 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+### Steam Deck
+
+#### Copypaste script:
+
+* Switch to Desktop mode (`STEAM` Button > Power > Switch to Desktop)
+* Open `Konsole` (Steam Deck Icon in bottom left corner > System > Konsole)
+* Copy the script, paste it in the CLI and press Enter("A" button on Steam Deck) 
+
+```
+[ ! -d ~/.local/bin ] && mkdir ~/.local/bin && echo "export $PATH=$HOME/.local/bin:$PATH" >> ".$(echo $SHELL | cut -d '/' -f 3)rc"
```
Comment (justchokingaround) [inline README.md:216]: use
```sh
".$(echo $SHELL | sed -nE "s|.*/(.*)\$|\1|p")rc"
```
instead, it's less error prone

File: README.md [hunk 6]
```diff
@@ -203,6 +204,93 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+~/.fzf/install
+
+mkdir ~/.aria2c
+wget -O ~/.aria2c/aria2-1.36.0.tar.bz2 https://github.com/q3aql/aria2-static-builds/releases/download/v1.36.0/aria2-1.36.0-linux-gnu-64bit-build1.tar.bz2
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+cp ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.local/bin/
+chmod +x ~/.local/bin/aria2c
+
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd
```
Comment (justchokingaround) [inline README.md:228]: last command is useless: `cd`

File: README.md [hunk 7]
```diff
@@ -203,6 +204,93 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+cp ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.local/bin/
+chmod +x ~/.local/bin/aria2c
+```
+
+##### Install ani-cli:
+
+```
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd
```
Comment (justchokingaround) [inline README.md:273]: last command is useless: `cd`

File: README.md [hunk 8]
```diff
@@ -203,6 +204,93 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
...
+```
+git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
+~/.fzf/install
+```
+press enter("A" button on Steam Deck) on questions
+
+##### Make a ~/.local/bin folder if doesnt exist and add it to $PATH
+
+```
+[ ! -d ~/.local/bin ] && mkdir ~/.local/bin && echo "export $PATH=$HOME/.local/bin:$PATH" >> ".$(echo $SHELL | cut -d '/' -f 3)rc"
```
Comment (justchokingaround) [inline README.md:256]: use
```sh
".$(echo $SHELL | sed -nE "s|.*/(.*)\$|\1|p")rc"
```
instead, it's less error prone

**Review by justchokingaround — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Nannk) [reply README.md:228]: cant say its useless here
i need to get back to ~, right?

**Review by justchokingaround — APPROVED** [review]

**Pull request merged**